### PR TITLE
docs/getting-started-stock-update

### PR DESCRIPTION
**Pull request by kamil-musialowski**
Improved [Getting started with Stock docs](https://www.highcharts.com/docs/stock/getting-started-stock).

### CODE REVIEW

File: docs/stock/getting-started-stock.md [hunk 1]
```diff
@@ -1,49 +1,63 @@
...
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/stock/modules/stock.js"></script>
+```
 For alternative loading and bundling patterns, for UMD, AMD, CommonJS or ES6 modules, find more information [here](https://github.com/highcharts/highcharts/blob/master/readme.md). Highcharts Stock follows the same patterns as described for Highcharts.
 
 Constructor
 -----------
 
-Run the `stockChart` constructor for initializing a Stock chart visualization. The constructor takes two required parameters and a third optional parameter.
+Run the `stockChart` constructor for initializing a Stock chart visualization. The constructor takes two required parameters (container ID and a config object) and a third optional parameter, which is the callback function ran after the chart has loaded.
```
Comment (DJTechnoo) [inline docs/stock/getting-started-stock.md:27]:
```suggestion
Run the `stockChart` constructor for initializing a Stock chart visualization. The constructor takes two required parameters (container ID and a config object) and a third optional parameter, which is the callback function run after the chart has loaded.
```

File: docs/stock/getting-started-stock.md [hunk 2]
```diff
@@ -1,49 +1,63 @@
...
 
-[See also explained here in Stock API](https://api.highcharts.com/class-reference/Highcharts#.stockChart/)
+[See also explained here in Stock API](https://api.highcharts.com/class-reference/Highcharts.StockChart)
 
-Simple example
+Basic example
 --------------
 
-_See below the simple live example of a Stock chart_
+To create your first simple stock chart, all you need to do is to define the dataset appropriate for the [series type](https://api.highcharts.com/highstock/series) that you choose. The default series type is a [line series](https://api.highcharts.com/highstock/series.line).
```
Comment (DJTechnoo) [inline docs/stock/getting-started-stock.md:44]: "Simple" should maybe also be changed to "basic" here as well

**Review by DJTechnoo — CHANGES_REQUESTED** [review]
This looks much better! Just some small nit-picking from my side

File: docs/stock/getting-started-stock.md [hunk 3]
```diff
@@ -1,49 +1,63 @@
...
 
-Run the `stockChart` constructor for initializing a Stock chart visualization. The constructor takes two required parameters and a third optional parameter.
+Run the `stockChart` constructor for initializing a Stock chart visualization. The constructor takes two required parameters (container ID and a config object) and a third optional parameter, which is the callback function ran after the chart has loaded.
 
-    
-    Highcharts.stockChart('container', {
-        // configuration options
-    }, myCallback); 
+```js
+Highcharts.stockChart('container ID', {
```
Comment (pawelfus) [inline docs/stock/getting-started-stock.md:30]: According to the HTML 4/5 spec ID's must not contain empty spaces

**Review by pawelfus — CHANGES_REQUESTED** [review]

File: docs/stock/getting-started-stock.md (hunk 1 above)
Comment (kamil-musialowski) [reply docs/stock/getting-started-stock.md:27]: Nice catch, thanks!

**Review by DJTechnoo — APPROVED** [review]

**Review by bm64 — APPROVED** [review]

File: docs/stock/getting-started-stock.md [hunk 4]
```diff
@@ -1,49 +1,63 @@
...
-        // configuration options
-    }, myCallback); 
+```js
+Highcharts.stockChart('containerID', {
+    // configuration options object
+}, callback);
+```
 
-1.  `id:` The `id` of the HTML element used for rendering the chart.
+1.  `containerID:` The `id` of the HTML element used for rendering the chart.
```
Comment (pawellysy) [inline docs/stock/getting-started-stock.md:35]: or the element itself?

File: docs/stock/getting-started-stock.md [hunk 5]
```diff
@@ -1,49 +1,63 @@
...
 
-Run the `stockChart` constructor for initializing a Stock chart visualization. The constructor takes two required parameters and a third optional parameter.
+Run the `stockChart` constructor for initializing a Stock chart visualization. The constructor takes two required parameters (container ID and a config object) and a third optional parameter, which is the callback function run after the chart has loaded.
 
-    
-    Highcharts.stockChart('container', {
-        // configuration options
-    }, myCallback); 
+```js
+Highcharts.stockChart('containerID', {
```
Comment (pawellysy) [inline docs/stock/getting-started-stock.md:30]:
```suggestion
Highcharts.stockChart(containerID' {
```

**Review by pawellysy — CHANGES_REQUESTED** [review]
minor polishing

File: docs/stock/getting-started-stock.md (hunk 5 above)
Comment (pawelfus) [reply docs/stock/getting-started-stock.md:30]: Hm.. something strange happened here :D

File: docs/stock/getting-started-stock.md (hunk 3 above)
Comment (pawelfus) [reply docs/stock/getting-started-stock.md]: Yes, that sounds good.

**Review by pawelfus — APPROVED** [review]
Thanks!

**Review by pawellysy — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**